recipes/gym/meta.yaml

```diff
@@ -69,7 +78,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage('gym', exact=True) }}
-        - atari_py ==0.2
```

Comment:
@oblute
Is there a reason you removed pins for atari_py and box2d-py? These are from the original requirements (like we're also constricting the ranges of other run requirements)

Reply:
@h-vetinari the pins aren't required, given that those are the only versions of those packages in conda-forge currently, and the specific pins would only prevent this working for any newer, future versions released of them.

Reply:
I understand that, but I would still reflect the upstream position on this; possibly they want to specifically lock this in (like for `cloudpickle`, `pyglet`), to prevent unexpected API breakage.

Alternatively, we could leave the dependencies unpinned (and with no upper limit in the run-reqs), but run the test suite as part of testing the build.

WDYT?

Reply:
@h-vetinari good point about the potential for locking it in; i think we should add the pins back for these two. pushing now.